public/docs/_examples/structural-directives/ts/src/app/app.component.html

```diff
@@ -152,11 +152,9 @@ <h2 id="ngSwitch">NgSwitch</h2>
 
 <div>Pick your favorite hero</div>
 <p>
-  <ng-container *ngFor="let h of heroes">
-    <label>
-      <input type="radio" name="heroes" [(ngModel)]="hero" [value]="h">{{h.name}}
-    </label>
-  </ng-container>
+  <label *ngFor="let h of heroes">
```

Comment:
Moving the ngFor to the label makes a lot of sense and we should do it **UNLESS this is an example of `<ng-container>`**.  If it is, please restore. If it isn't, then your change is an improvement.